csharp-htk/obj/Debug/netstandard2.0/HTK.GeneratedMSBuildEditorConfig.editorconfig

```diff
@@ -0,0 +1,3 @@
+is_global = true
+build_property.RootNamespace = HTK
+build_property.ProjectDir = D:\Coding\csharp-htk\csharp-htk\
```

Comment:
I think this should potentially be `.gitignore`-d because the giveaway is that `D:\` is a unique path